test/services/matching-worker/matching-worker-process.js

```diff
@@ -80,7 +80,7 @@ const workerContext = {
 process.argv[2] = JSON.stringify(workerContext)
 const matchingWorkerProcess = require('../../../lib/fhir/services/matching-worker/matching-worker-process')
 
-tap.test('should execute processResourceScore() and return a object', (t) => {
+tap.test('should execute processResourceScore() and return a object', { skip: true }, (t) => {
```

Comment:
Are we skipping this test due to it failing for a random reason? and needs further investigation

Reply:
Yeah, they weren't actually being run because hearth was only running tests in the test 1 level down. So with my change to the `package.json` they are actually failing and I'm not quite sure of a good way to fix them atm.

Reply:
ok cool makes sense to me to skip them for now as they were never run before, and currently breaking the tests